crates/apollo-compiler/src/validation/selection.rs

```diff
@@ -35,6 +313,10 @@ pub fn validate_selection_set(
 ) -> Vec<ApolloDiagnostic> {
     let mut diagnostics = Vec::new();
 
+    if let Err(diagnostic) = fields_in_set_can_merge(db, selection_set.clone()) {
```

Comment:
the nittiest nit in the world of nits! we've been doing `db.func_name`, which makes it easier to know visually that it's a db query. shall we stick to that?

```suggestion
    if let Err(diagnostic) = db.fields_in_set_can_merge(selection_set.clone()) {
```

Reply:
it might? am not sure